src/sage/graphs/generic_graph.py

```diff
@@ -938,6 +939,143 @@ def _latex_(self):
 
         return self.latex_options().latex()
 
+    def tikz(self, format='dot2tex', edge_labels=None,
+            color_by_label=False, prog='dot', rankdir='down',
+            use_sage_preamble=None, **kwds):
```

Comment:
you should say somewhere where to look for the extra optional parameters in `**kwds`.

Reply:
Done.